Stop relying on implicit usage of GLU for compability with >=x11-libs/qt-opengl-4.8
See, http://bugs.gentoo.org/407391

--- mpeg2window/ttmpeg2window.cpp
+++ mpeg2window/ttmpeg2window.cpp
@@ -33,6 +33,7 @@
 // qApplication for the qApp pointer
 #include <QApplication>
 #include <QPixmap>
+#include <GL/glu.h>
 
 // -----------------------------------------------------------------------------
 // class declaration
--- ttcut.pro
+++ ttcut.pro
@@ -15,7 +15,7 @@
 DEFINES     += _FILE_OFFSET_BITS=64
 DIST        = TTCut
 TARGET      = ttcut
-LIBS        = -lmpeg2 -lmpeg2convert
+LIBS        = -lmpeg2 -lmpeg2convert -lGLU
 OBJECTS_DIR = obj
 MOC_DIR     = moc
 UI_DIR      = ui_h 
